docs/getting-started/keeping-backstage-updated.md

```diff
@@ -0,0 +1,33 @@
+---
+id: keeping-backstage-updated
+title: Keeping Backstage Updated
+description: How to keep your Backstage App updated
+---
+
+Backstage is always improving, so it's a good idea to stay in sync with the
+latest releases. Backstage is more of a library than an application or service;
+similar to `create-react-app`, the `@backstage/create-app` tool gives you a
+starting point that's meant to be evolved.
+
+## Updating Backstage versions with backstage-cli
+
+The Backstage CLI has a command to bump all `@backstage` packages you're using
+to the latest versions:
+[versions:bump](https://backstage.io/docs/cli/commands#versionsbump).
+
+```bash
+npx backstage-cli versions:bump
+```
+
+The reason for bumping all `@backstage` packages at once is dependency between
+Backstage packages. React Context, for example, may not be referentially equal
+if multiple versions of `@backstage/core` are loaded.
```

Comment:
The sentence about React Context is true, but I think that it's usually not the cause for somebody bumping. It's rather a symptom that usually manifests itself when yarn is being silly, in general. I would vote to make an entire heading about the `versions:check` command (and its `--fix` flag) at the bottom of this very file, that mentions that problem in the context of (hah!) react contexts and other problems that can happen if you have multiple versions of central packages present.